client/src/components/exercises/ExerciseItem/index.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { FormControlLabel, Checkbox, Typography } from '@material-ui/core';
+import AddCircleOutlinedIcon from '@material-ui/icons/AddCircleOutlined';
+import { func, shape, bool } from 'prop-types';
+
+function ExerciseItem(props) {
+  const { handleClickOpen, element, isChecked } = props;
+  const {
+    exercise_name: exerciseName,
+    caloriesPerHour,
+    id: exerciseId,
+  } = element;
+  return (
+    <div
+      style={{
+        display: 'flex',
+        justifyContent: 'space-around',
+        width: '80vw',
+        alignItems: 'center',
+        border: '1px solid black',
+        borderRadius: '1rem',
+        padding: '0.5rem',
+        '@media(minWidth:600px)': {
+          width: '40vw',
```

Comment:
Here’s another place where separating out your styles will make the code cleaner (because you’re not distracted by them when trying to understand the purpose or behavior of this component) and the jsx shorter.